Assets/Scripts/Webcam/UnityWebcam.cs

```diff
@@ -94,5 +94,10 @@ public float GetRotation()
 			}
 			return rotation;
 		}
+
+		public int GetCRC()
```

Comment:
I prefer "Checksum" than "CRC" because CRC is a specific type of checksum. If i am not mistaken, this is not a CRC.